### ISSUE TITLE
Colorable Accessories Bug.

### ISSUE THREAD

**Issue by VeteranGary** [Aurorastation/Aurora.3#12200]
Round ID: cc4-cXc1
Bug: Certain colorable clothing accessories were locked to a dark black color, and cannot be changed no matter what is done; reloading the slot, unselecting the accessory and picking a new one, etc. Not sure how to recreate the issue, but I thought it was a graphical error, such as an update to the accessories. It was not, I cleared my cache and relogged, and other players could see my character's accessory appearing a dark black. Here in this photo, I tried setting it to bright white, and the accessory is still a dark black oddly enough. I don't see any changelogs making the dress shirts black by default, and for testing I swapped to another character slot and the coloring worked perfectly fine. 
Bugged slot: https://i.gyazo.com/9d5639459e7ab86be5eeac58683dcacb.png
Working slot: https://i.gyazo.com/ec005df263e23200ee865e5bedca5489.png

Tl;dr, all the custom shirt accessories are stuck in this dark black color, can only be made darker or shaded with other colors, but cannot be any lighter. It seems to be effecting only one of my character slots, not all of them. It is also effecting other items like the SCC armband oddly enough, which appears as pitch black.
e: any accessory added also will appear as black, such as any armband. it will force the item to appear pitch black, even if the item is not supposed to be colorable. This is also visible ingame, and not a clientside error, other players can see the issue.
edit part 2: messing around a bit further, i've discovered a possible issue. Any accessory attached to the colorable pants item seems to be forced to match the color set to that, so shirts, ties, armbands, all will get overrided to match that. Have not tested on other character slots to see if the issue is replicable.
